sections/image-with-text.liquid

```diff
@@ -27,22 +27,24 @@
         {%- endif -%}
       </div>
     </div>
-    <div class="grid__item">
-      <div class="image-with-text__content image-with-text__content--{{ section.settings.height }}">
+    <div class="image-with-text__text-item grid__item">
+      <div id="ImageWithText--{{ section.id }}" class="image-with-text__content image-with-text__content--{{ section.settings.desktop_content_position }} image-with-text__content--desktop-{{ section.settings.desktop_content_alignment }} image-with-text__content--mobile-{{ section.settings.mobile_content_alignment }} image-with-text__content--{{ section.settings.height }} gradient color-{{ section.settings.color_scheme }}">
         {%- for block in section.blocks -%}
           {% case block.type %}
             {%- when 'heading' -%}
-              <h2 class="image-with-text__heading h1" {{ block.shopify_attributes }}>
+              <h2 class="image-with-text__heading {{ block.settings.heading_size }}" {{ block.shopify_attributes }}>
                 {{ block.settings.heading | escape }}
               </h2>
+            {%- when 'subtitle' -%}
+              <div class="image-with-text__text {{ block.settings.text_style }}" {{ block.shopify_attributes }}>{{ block.settings.subtitle }}</div>
```

Comment:
Since this is input by the merchant and is not possible to edit by the buyer, I think it's secure to not escape it, but it does open up a wider conversation around whether or not `| escape` should be needed for settings of type `text`.  

Since for something like a custom liquid section we use a `liquid` setting type, which is explicitly for code, we might want to consider automatically escaping `text` inputs and either having folks use the `liquid` setting type or a `script` setting type that allows for JS.

There may be backwards compatibility issues with this.